src/Symfony/Bridge/Twig/Mime/BodyRenderer.php

```diff
@@ -77,6 +77,23 @@ public function render(Message $message): void
         $message->context($message->getContext() + [__CLASS__ => $currentRenderingKey]);
     }
 
+    private function getFingerPrint(TemplatedEmail $message): string
+    {
+        $messageContext = $message->getContext();
+        unset($messageContext[__CLASS__]);
+
+        $payload = [$messageContext, $message->getTextTemplate(), $message->getHtmlTemplate()];
+        try {
+            $serialized = serialize($payload);
+        } catch (\Exception $e) {
+            // Serialization of 'Closure' is not allowed
+            // Happens when context contain a closure, in that case, we assume that context always change.
+            $serialized = uniqid('', true);
```

Comment:
`random_bytes(8)` would be better (8 or anything else) :)

Reply:
done